zerver/lib/actions.py

```diff
@@ -1215,16 +1215,18 @@ def set_stream_color(user_profile, stream_name, color=None):
     return color
 
 def get_subscribers_to_streams(streams, requesting_user=None):
-    # type: (Iterable[Stream], Optional[UserProfile]) -> Dict[UserProfile, List[Stream]]
-    """ Return a dict where the keys are user profiles, and the values are
+    # type: (Iterable[Stream], Optional[UserProfile]) -> Dict[int, List[Stream]]
+    """ Return a dict where the keys are user profile id, and the values are
     arrays of all the streams within 'streams' to which that user is
     subscribed.
     """
     # TODO: Modify this function to not use UserProfile objects as dict keys
```

Comment:
You can remove this TODO now since it looks like you've changed the dict to use profile ids as keys instead